x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_to_dashboard_drilldown/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EmbeddableContext,
+  ValueClickTriggerContext,
+  RangeSelectTriggerContext,
+  IEmbeddable,
+} from '../../../../../../../src/plugins/embeddable/public';
+
+export type PlaceContext = EmbeddableContext;
+export type ActionContext<T extends IEmbeddable = IEmbeddable> =
```

Comment:
I see this type now, can't find my previous comment (I so wish github handled reviews on large PRs better).

This is confusing because of the other `ActionContext` type.  I think it shouldn't be generic, since it should work on any embeddable, not the visualize one specifically.

Reply:
Agreed, see comment above https://github.com/elastic/kibana/pull/61219/#discussion_r417882060